addons/website/models/website_visitor.py

```diff
@@ -109,6 +109,13 @@ def _compute_page_statistics(self):
             visitor.visitor_page_count = visitor_info['visitor_page_count']
             visitor.page_count = visitor_info['page_count']
 
+    def _search_page_ids(self, operator, value):
+        if operator not in ('like', 'ilike', 'not like', 'not ilike', '=', '!='):
```

Comment:
Probably those 2 at that point?
```suggestion
        if operator not in ('like', 'ilike', 'not like', 'not ilike', '=like', '=ilike', '=', '!='):
```